_config.yml

```diff
@@ -1,34 +1,38 @@
 # Permalinks
 #
 # Use of `relative_permalinks` ensures post links from the index work properly.
-permalink:           pretty
+permalink: pretty
 
 # Setup
-title:               Operação Serenata de Amor
-tagline:             Robôs contra a corrupção
-url:                 https://serenatadeamor.org
-baseurl:             ""
+title: Operação Serenata de Amor
+tagline: Inteligência artificial para controle social da administração pública
+url: https://www.serenatadeamor.org
+baseurl: ""
 
 # Assets
 #
 # We specify the directory for Jekyll so we can use @imports.
 sass:
-  sass_dir:          _sass
-  style:            :compressed
+  sass_dir: _sass
+  style: :compressed
 
 # About/contact
 author:
-  name:              Data Science Brigade
-  url:               https://www.facebook.com/operacaoSerenataDeAmor
-  email:             contato@datasciencebr.com
+  name: Operação Serenata de Amor
+  url: https://www.serenatadeamor.org
+  email: op.serenatadeamor@gmail.com
```

Comment:
`contato@serenata.ai`?

Reply:
Can we wait the official release of the new domain? I couldn't find this email in any public and official page.

Reply:
> I couldn't find this email in any public and official page.

Cyclical, hum? If we never update our official pages, we will never find it in the official pages hahaha…